src/chrome/content/rules/cloudcma.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Invalid certificate:
+		www.agent.cloudcma.com
+		www.partner.cloudcma.com
+		www.powerpack.cloudcma.com
+		www.retech.cloudcma.com
+		www.tomferry.cloudcma.com
+		webinar.cloudcma.com
+		www.webinar.cloudcma.com
+		www.ztc.cloudcma.com
```

Comment:
Wildcard DNS record

Reply:
What do I need to do? Remove them?

Reply:
Please add a comment similar to https://github.com/EFForg/https-everywhere/search?q=wildcard+dns